lib/src/widgets/movie_detail_image.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import '../../utils/constants.dart';
+
+class MovieDetailImage extends StatelessWidget {
+  final String posterPath;
+  final BoxFit imageBoxFit;
+
+  MovieDetailImage({
+    required this.posterPath,
+    this.imageBoxFit = BoxFit.fill,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(
+        Constants.paddingImage,
+      ),
+      child: FadeInImage.assetNetwork(
+        placeholder: posterPath,
+        image: posterPath,
```

Comment:
Why you use placeholder and image with the same path?

Reply:
You can add a specific route to placeholder, with another image that don't have relation with posterPath that you want to show

Reply:
Because the default image already came in posterPath